src/main/java/org/jfrog/gradle/plugin/artifactory/listener/ProjectsEvaluatedBuildListener.java

```diff
@@ -85,6 +87,7 @@ public void afterEvaluate(Project project) {
             if (!(task instanceof ArtifactoryTask)) {
                 return;
             }
+            addResolver(project);
```

Comment:
It looks like adding the resolver value is not dependent on the task and can repeat multiple times. Let’s move this line outside the loop.